index.html

```diff
@@ -289,6 +289,36 @@ <h2 class="header-text">About <br>Myself</h2>
       </div>
     </div>
   </section>
+  <section id="Contact" class="contact-me">
+    <h2 class="intro">Contact me</h2>
+    <p class="intro-contact">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it. </p>
+    <form action="https://formspree.io/f/mlekbpkb" method="POST">
+      <ul>
```

Comment:
- [ ] Nice work you did so far 👏 . However, Kindly use flexbox to layout elements of the form filed. This give you extra flexibility 👍 

## Instructions :
![image](https://user-images.githubusercontent.com/50990925/220591752-03103517-3a1b-49b1-9941-fb2e6961acf7.png)